src/test/java/org/folio/search/service/browse/CallNumberBrowseResultConverterTest.java

```diff
@@ -161,7 +161,8 @@ private static Stream<Arguments> testDataProvider() {
         forwardContext(), true, List.of(
           cnBrowseItem(instance("B1", "B2", "C2"), "B1"),
           cnBrowseItem(instance("B1", "B2", "C2"), "B2"),
-          cnBrowseItem(instance("C1", "C2"), "C1"))),
+          cnBrowseItem(instance("C1", "C2"), "C1"),
+          cnBrowseItem(2, "C2"))),
```

Comment:
5 string above you can see that we have `instance("B1", "B2", "C2")` and `instance("C1", "C2")`
So, in the result we should have 2 instances with same call-number 'C2'. 
Before it was cut off, because `upper` value was 'C1'